roles/davinci_resolve/tasks/main.yml

```diff
@@ -108,6 +109,13 @@
     creates: "/tmp/{{ product | replace(' ', '_') }}_{{ pkgver }}.zip"
     follow_redirects: all
     status_code: 200
+  when: davinci_resolve_zip_file is undefined
+
+- name: Use pre-installed DaVinci Resolve
+  ansible.builtin.copy:
```

Comment:
As ansible-lint commented, it is better to explicitly set the file permission: https://ansible.readthedocs.io/projects/lint/rules/risky-file-permissions/#correct-code